integration/tests/bft/topicvalidator/orchestrator.go

```diff
@@ -31,11 +31,7 @@ const (
 
 // TopicValidatorAttackOrchestrator represents an insecure.AttackOrchestrator track incoming unauthorized messages and authorized messages received by victim nodes.
 type TopicValidatorAttackOrchestrator struct {
```

Comment:
```suggestion
type Orchestrator struct {
```
Naming convention in Golang implies that module name should not start with the name of package.
> According to Package Names at golang.org, all references to names in a package will be done using the package name, so one can omit that name from the identifiers. For example, if you are in a package foo, you do not need a type FooFile, which clients will write as foo.FooFile. Instead, we name the type File, which clients will write as foo.File.

Reply:
https://github.com/onflow/flow-go/pull/3877/commits/912800fdd5d723ef5ca23a4b6a53fe7e94c1c6c4